release_notes/v2.2.8.txt

```diff
@@ -0,0 +1,15 @@
+Release Notes
+-------------
+This v2.2.8 version is the latest of the v2.2 long-term support (LTS) release
+of the fabric-node-sdk packages that support Fabric 2.x, and supersedes previous
+v2.x releases. Please see the Fabric LTS release strategy document for more
+information on LTS releases:
+
+https://github.com/hyperledger/fabric-rfcs/blob/main/text/0005-lts-release-strategy.md
+
+Changes in this release:
+
```

Comment:
Do you want to mention the hang fix so that people using v2.2.6 and v2.2.7 will know to upgrade?